src/targets/cpu/lowering.cpp

```diff
@@ -43,6 +43,8 @@
 #include <migraphx/op/argmax.hpp>
 #include <migraphx/op/argmin.hpp>
 #include <migraphx/op/rnn_var_sl_last_output.hpp>
+#include <migraphx/op/mod.hpp>
+#include <migraphx/op/fmod.hpp>
```

Comment:
Are there commits to be pushed yet?

Reply:
No these should already be in the build.